app/components/attachment_button/index.js

```diff
@@ -278,7 +273,9 @@ export default class AttachmentButton extends PureComponent {
                 res.uri = decodeURIComponent(res.uri);
 
                 this.uploadFiles([res]);
-            });
+            } catch (error) {
+                // Do nothing
```

Comment:
out of scope, but please create a ticket so if this fails we can show an Alert so the user knows that something went wrong attaching the file